### ISSUE TITLE
Please add Instructions for CSV to landing page

### ISSUE THREAD

**Issue by mrl2718** [prisma-flowdiagram/PRISMA2020#65]
I had to pull up the Preprint in order to learn how to use the CSV (i.e., which columns to edit vs leave untouched). ... I never found explanation that to turn off a box, I had to delete all text from the "n" column; I found that out by trial and error.

A few simple instructions, and perhaps a worked example, would be very helpful.

This is a lovely tool - works great - thank you for creating and sharing!

**Comment by silas-blomqvist** [comment]
@mrl2718 How did you edit the csv to produce the flowchart in r? Whenever I try to enter something in the n column I get an error (Error in if (is.na(previous_studies) == TRUE && is.na(previous_reports) ==  : 
  missing value where TRUE/FALSE is demanded) when reading the csv file.

**Comment by chriscpritchard** [comment]
@silas-blomqvist  - perhaps you could send me the CSV file you're trying to use? I don't get that error!

**Comment by juliandehne** [comment]
I agree that the input is not that clear, instead of an CSV, an R object should be admissable.
